app/models/user.rb

```diff
@@ -14,6 +15,9 @@ class User < ApplicationRecord
   after_destroy :ensure_an_admin_remains
   after_create_commit :send_welcome_mail
 
+  has_many :orders, dependent: :destroy
+  has_many :line_items, through: :orders
```

Comment:
follow this order
associations  -> validations -> callbacks

Reply:
done